docs/datasets_and_tables/views.md

```diff
@@ -1,45 +1,33 @@
-|gtfs_schedule_fact_day_of_week_service_monthly_comparison|           |[link](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.gtfs_schedule_fact_day_of_week_service_monthly_comparison)|
-|gtfs_schedule_fact_route_id_changes                      |           |[link](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.gtfs_schedule_fact_route_id_changes)                      |
-|gtfs_schedule_fact_stop_id_changes                       |           |[link](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.gtfs_schedule_fact_stop_id_changes)                       |
-|gtfs_schedule_index_feed_trip_stops                      |           |[link](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.gtfs_schedule_index_feed_trip_stops)                      |
-|gtfs_schedule_stg_calendar_long                          |           |[link](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.gtfs_schedule_stg_calendar_long)                          |
-|gtfs_schedule_stg_daily_service                          |           |[link](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.gtfs_schedule_stg_daily_service)                          |
-|reports_gtfs_schedule_index                              |           |[link](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.reports_gtfs_schedule_index)                              |
-|reports_weekly_file_checks                               |           |[link](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.reports_weekly_file_checks)                               |
-|validation_code_descriptions                             |           |[link](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.validation_code_descriptions)                             |
-|validation_dim_codes                                     |           |[link](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.validation_dim_codes)                                     |
-|validation_fact_daily_feed_codes                         |           |[link](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.validation_fact_daily_feed_codes)                         |
-|validation_fact_daily_feed_notices                       |           |[link](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.validation_fact_daily_feed_notices)                       |
+(view-models)=
+# Views
+
+End-user-friendly data for dashboards and metrics
+
+## How to view the `views` documentation
+
+Visit the [dbt Cal-ITP warehouse documentation](https://dbt-docs.calitp.org/#!/overview).
+
+For help navgating the documentation, visit [Navigating the dbt Docs](navigating-dbt-docs).
+
+## Database Schemas
+
+### Views
+| dataset name |
+| ------- |
+| `views.*` |
+| `mart_transit_database.*` |
+
+### Views Staging
```

Comment:
I would actually remove this I think? Alternatively, would need to include Payments views staging, no?

Reply:
@lauriemerrell yeah I'm going to remove now

Reply:
@lauriemerrell I did add it to the `GTFS Schedule` doc though, as it looks like it's mostly just cleaned schedule data? let me know if I should remove

Reply:
Yeah, that's ok